routes/api/hacker.js

```diff
@@ -222,7 +222,7 @@ module.exports = {
          * @apiGroup Hacker
          * @apiVersion 0.0.9
          *
-         * @apiParam (body) {string} [status] Status of the hacker's application ("None"|"Applied"|"Waitlisted"|"Confirmed"|"Cancelled"|"Checked-in")
+         * @apiParam (body) {string} [status] Status of the hacker's application ("None"|"Applied"|"Waitlisted"|"Accepted"|"Declined"|"Confirmed"|"Withdrawn"|"Checked-in")
```

Comment:
nit: alphabetical or grouped by stage of status

Reply:
Made it: None, Applied. Then Accepted Declined Waitlisted. Then Confirmed Withdrawn. Then Checked In. Grouped by stage + alphabetical within.